### PR TITLE
Compact search improvements

### PR DESCRIPTION
**Pull request by akhmorgan**

### CODE REVIEW

File: src/sidebars/sidebar.less [hunk 1]
```diff
@@ -100,20 +100,25 @@
...
     background: transparent;
   }
+  .favourites {
+    margin-top: 34px;
+  }
   .favourites, #favScroll, .favourite {
     width: 100%;
   }
   .search {
+    transition: all 0.1s linear 0.2s;
```
Comment (owennw) [inline src/sidebars/sidebar.less]: This feels a bit too fast; can you use the `@transition` variable instead?

File: src/sidebars/search/search-controller.js [hunk 2]
```diff
@@ -32,6 +32,10 @@
                 // Down
                 this.changePointer(1);
             }
+            else if (event.keyCode === 27) {
+                // Escape
+                this.$scope.sidebarCtrl.favouritesClick();
+            }
         }
```
Comment (owennw) [inline src/sidebars/search/search-controller.js]: This also applies in the main view. However, I think I'm okay with that.

**Comment by owennw** [comment]
There's a very small hangover for when the app is expanding -- you can see the favourites tab grow vertically upwards. Besides that all good!

File: src/sidebars/sidebar.less (hunk 1 above)
Comment (akhmorgan) [inline src/sidebars/sidebar.less]: Yeah that's fine

**Pull request merged**